extensions/tangleid/main.py

```diff
@@ -175,3 +181,20 @@ def new_group(data):
     response = send_transfer(tag, json.dumps(data), address, 0, dict_tips, debug=0)
 
     return str(response)
+
+def get_all_claims_in_channel(data):
```

Comment:
Don't touch TangleID extension while implementing core routines about IPM. Instead, you shall get it done as the top level service.